lib/bloc/world_clock_state.dart

```diff
@@ -1,13 +1,13 @@
-class WorldClockState{}
+import '../models/world_clock_model.dart';
 
-class WorldClockDataState extends WorldClockState{
-   final List<DateTime> timeData;
+class WorldClockState {}
```

Comment:
base `State/Event` is better to be `abstract class` if you're using them for inheritance purposes only